jks/certbot-renew-hook.sh

```diff
@@ -22,6 +22,16 @@ if [ -r "${CERTDIR}/keystore.jks" ] ; then
     echo "Updating java keystore file for ${CERTNAME}"
     DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
     ${DIR}/update-jks -f "${CERTNAME}" $(head -n 1 "${CERTDIR}/keystore.jks.pwd") "${CERTDIR}/keystore.jks"
+    echo "Deploying secrets referencing '${CERTDIR}/keystore.jks'"
+    find -L /etc/apifocal/ -samefile "${CERTDIR}/keystore.jks" | \
+	{
+	    while read SECRET
+	    do
+		SECRETNAME=$(echo "${SECRET}" | cut -d '/' -f 6 )
```

Comment:
I'd rather compute the numbers in `cut` based on the root folder `/etc/apifocal` - I think this will create silent errors later on, if we somehow migrate certs to a different location/folder etc